system/Debug/Toolbar.php

```diff
@@ -258,19 +379,38 @@ public static function eventHandler()
 		{
 			helper('security');
 
+			$format = $request->negotiate('media', [
+				'text/html',
+				'application/json',
+				'application/xml'
+			]);
+			$format = explode('/', $format)[1];
+
 			$file     = sanitize_filename('debugbar_'.$request->getGet('debugbar_time'));
 			$filename = WRITEPATH.'debugbar/'.$file;
 
 			// Show the toolbar
 			if (file_exists($filename))
 			{
-				$contents = file_get_contents($filename);
-				unlink($filename);
+				$contents = self::format(file_get_contents($filename), $format);
+				//unlink($filename); // TODO - Keep history? 10 files
```

Comment:
At the moment the files are not being deleted, so that tests can be done by accessing the URLs directly.

Thinking about the possibility of maintaining a history of requests. Initially just keeping some files that can be handled later.

Reply:
Having a history, besides being able to show it in the debug bar, also could be possible add a custom Header or append a link to the debug data.

I was thinking about being able to standardize API responses. For example:

```json
{
	"meta":
	{
		"code": 200,
		"pagination":
		{
			"next": ""
		}
	},
	"data": {},
	"debug": {}
}
```

Since the the _response body_ is inside the _data_. It would be nice to manipulate debugbar and already show some data by default. But this could overfill the output.

Reply:
#863

Reply:
http://jsonapi.org/